backend/lib/endpoints/users.js

```diff
@@ -37,6 +37,17 @@ async function routes(app) {
       };
     },
   );
+
+  app.post(
+    "/signup/createProfile",
```

Comment:
Another thing with `signup/createProfile` as a route
usually with REST it's a convention to use resources as the endpoints and use HTTP verbs, but avoid naming endpoints as actions such as "create..." etc.